docs/get-started/git/git-basics.md

```diff
@@ -0,0 +1,95 @@
+---
+sidebar_position: 1
+title: "Git Basics"
+---
+
+## Selected Git Features
+
+This brief introduction to Git uses the command line to illustrate some of its features. If you want to try it out, download and install Git on your laptop or workstation: [git-scm.com](https://git-scm.com/).
+
+### Repositories
+
+A repository is a place where all of the changes in a project are stored. In the simplest case this is a folder named .git in the same folder as the project source files. To get started, create an empty folder, make it the current directory with `cd` and then try this command:
+
+```
+git init .
+```
+
+which should result in confirmation that an empty Git repository has been created in a hidden folder named .git (folders starting with a period(.) are usually left out of listings so you don't see them, hence hidden). 
+
+You may also use a remote repository on another computer, which is where GitHub comes in. That will be covered in separate articles.
+
+Create some content with a text editor and save it in the source folder, the one containing the .git repo (the word repo is often used as a short form of repository). In this article this markdown document, aptly named git.md, is used as an example document. Confirm what you have with a list command:
+
+```
+ls -al
```

Comment:
this is a *nix only command and does not work on windows in either powershell or regular command line